npm-shrinkwrap.json

```diff
@@ -67,6 +67,36 @@
                                            "resolved":  "https://registry.npmjs.org/bootstrap/-/bootstrap-3.3.7.tgz",
                                            "Hash":  "6A5E17A69ADA3DFF645735129A614A5B2DEA23C519E5DD24ECB3A429BDE896EF"
                                        },
+                         "chart.js":  {
+                                          "version":  "2.7.0",
+                                          "from":  "https://registry.npmjs.org/chart.js/-/chart.js-2.7.0.tgz",
+                                          "resolved":  "https://registry.npmjs.org/chart.js/-/chart.js-2.7.0.tgz",
+                                          "Hash":  "700AC20CBC44BD7851AA5A5A064743D88675C9FEB2EFBF2796AED492359B838B"
+                                      },
+                         "chartjs-color":  {

```

Comment:
Let's make sure all the transitive dependencies are kosher to use as well

Reply:
All of the transitive dependencies are MIT.